pkg/apis/experimental/deep_copy_generated.go

```diff
@@ -986,6 +986,62 @@ func deepCopy_experimental_HorizontalPodAutoscalerStatus(in HorizontalPodAutosca
 	return nil
 }
 
+func deepCopy_experimental_IngressPoint(in IngressPoint, out *IngressPoint, c *conversion.Cloner) error {
+	if err := deepCopy_api_TypeMeta(in.TypeMeta, &out.TypeMeta, c); err != nil {
```

Comment:
for future reference, I'd split this out into 2 commits. The first containing just the resource and the second with all the auto generated boiler plate.